ide/api.debugger/src/org/netbeans/api/debugger/ActionsManager.java

```diff
@@ -164,7 +164,7 @@ public final void doAction (final Object action) {
     /**
      * Post action on this DebuggerEngine.
      * This method does not block till the action is done,
-     * if {@link #canPostAsynchronously} returns true.
+     * if <code>canPostAsynchronously</code> returns true.
```

Comment:
better @link to the method ?

Reply:
method is not existing. Not sure on how to rewrite

Reply:
// cc: @entlicher   ?